packages/action-menu/test/action-menu-directive.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { expect, oneEvent } from '@open-wc/testing';
+import { ActionMenu } from '@spectrum-web-components/action-menu';
+import { sendKeys } from '@web/test-runner-commands';
+
+import { directive } from '../stories/action-menu.stories.js';
+import { fixture } from '../../../test/testing-helpers.js';
+import { nextFrame } from '@spectrum-web-components/overlay/src/AbstractOverlay.js';
+
+describe('Slottable Request Directive', () => {
+    it('Action Menu requests for options rendering when opening and closing', async function () {
```

Comment:
A great addition would be if we can also include some tests where we leverage keyboard press events alongside the click events?